scripts/zones/Qufim_Island/mobs/Atkorkamuy.lua

```diff
@@ -4,8 +4,25 @@
 -----------------------------------
 local entity = {}
 
+entity.onMobInitialize = function(mob)
+    mob:setMobMod(xi.mobMod.ADD_EFFECT, 1)
+    mob:setMod(xi.mod.DOUBLE_ATTACK, 10)
```

Comment:
Fun little fact on this, Sea Monks are actually WAR/WAR not MNK jobs. This needs to be changed through the entire codebase (not in this PR of course). See video for evidence with attack rounds. This will also remove the need for the double attack in the lua. At a quick glance there is also no counter (as expected) -> but WAR/WAR should solve all your questions.

Reply:
Alright, so if they are meant to be WAR/WAR I can remove the DA being directly added for sure. Honestly, even on retail the DA didn't feel like it procced often so it being temporarily missing until sea monks themselves are fixed will likely barely be noticeable unless you are specifically looking for it.

Also, this explains why I was occasionally getting countered on my LSB fork which never happened on retail. It was infrequent enough I was thinking it could just be a sample size thing, but it not being MNK at all makes way more sense.

>the wiki's don't specify job or % of double attack
Yup, one wiki mentions DA, another mentions it MIGHT be a trait, and the others don't mention it at all. Like I initially said, the value in particular was picked based off of general feel + number of attacks between whirlwinds while not feeding any TP since I couldn't find anything concrete. 10% ended up being the value that felt the same as retail, which would also fit the trait if it is at least /WAR, and is only slightly under the trait value if it is WAR/WAR.

Reply:
> Fun little fact on this, Sea Monks are actually WAR/WAR not MNK jobs. This needs to be changed through the entire codebase (not in this PR of course). See video for evidence with attack rounds. This will also remove the need for the double attack in the lua. At a quick glance there is also no counter (as expected) -> but WAR/WAR should solve all your questions.
> 
> Some evidence of that here: https://www.youtube.com/watch?app=desktop&v=soQVWKwGHvc

I think there are one or two nm ones that are non war but yeah. Just having DA at all was evidense the job(s) were wrong. Delay od 360 still strikes me as unusual, people always try to estimate that and time it and get it wrong and the codebase has tons of old wrong values. Most non monk use 240

Reply:
> Alright, so if they are meant to be WAR/WAR I can remove the DA being directly added for sure. Honestly, even on retail the DA didn't feel like it procced often so it being temporarily missing until sea monks themselves are fixed will likely barely be noticeable unless you are specifically looking for it.
> 
> Also, this explains why I was occasionally getting countered on my LSB fork which never happened on retail. It was infrequent enough I was thinking it could just be a sample size thing, but it not being MNK at all makes way more sense.
> 
> >the wiki's don't specify job or % of double attack
> 
> Yup, one wiki mentions DA, another mentions it MIGHT be a trait, and the others don't mention it at all. Like I initially said, the value in particular was picked based off of general feel + number of attacks between whirlwinds while not feeding any TP since I couldn't find anything concrete. 10% ended up being the value that felt the same as retail, which would also fit the trait if it is at least /WAR, and is only slightly under the trait value if it is WAR/WAR.

Never do anything by feel. Delay can be figured out with tp gains but not DA. I suggest comparing it with the nearby chaemable kraken to see its tp per hit to math its delay from unless feank already has that as well

Reply:
I do not but there is this really nice chart on the monster stats sheet ;)
https://docs.google.com/spreadsheets/d/1YBoveP-weMdidrirY-vPDzHyxbEI2ryECINlfCnFkLI/edit#gid=1035487711

Reply:
So I didn't touch the delay or anything, that was all info that was previously there. The only things that went by feel were trying to get an idea of what the DA rate was like, and approximately how many hits before he whirlwinded at different amounts of HP. The attack delay and job info in the db is what was already set.

I did remove the addmod though, it sounded like the war/war change would be a sea monk family change as a whole though rather than just Atkorkamuy.

Reply:
You will have to change the job on the NM itself since it uses its own `mob_pool`

Reply:
Ah noticed the delay field in mob_pool as well, so should I set it to 240 then or wait for that to be fixed with an overall look at delays in general? Did get it set to war/war though

Reply:
> So I didn't touch the delay or anything, that was all info that was previously there

I am aware, github does a pretty good job of color highlight what was changed vs what was already there. I know I can be a little verbose in talking about how things "should be". I will often tell you _how_ to make things happen even if you aren't actively working on a thing, simply because its right there.

> Ah noticed the delay field in mob_pool as well, so should I set it to 240 then or wait for that to be fixed with an overall look at delays in general? Did get it set to war/war though

optional, but don't change it without verifying what it really is. My post had an edit and if reading from email you can easily miss that: I instructed how to find the accurate delay, and if you want to do it you can but its not required because delays nee a more general audit anyway (probably every sea monk in existence is wrong, but i haven't looked yet)